packages/thirdweb/src/react/web/ui/ConnectWallet/NetworkSelector.tsx

```diff
@@ -194,7 +201,8 @@ export function NetworkSelectorContent(props: NetworkSelectorContentProps) {
       ) {
         const recentChains = props.networkSelector.recentChainIds
           .map((id) => allChainsMap.get(id))
-          .filter((c) => c !== undefined);
+          .filter((c) => c !== undefined) as Chain[];
```

Comment:
this should not be needed with typescript `5.5`

Reply:
nice

Reply:
removed